Resources/public/js/views/ez-universaldiscoveryview.js

```diff
@@ -560,15 +561,15 @@ YUI.add('ez-universaldiscoveryview', function (Y) {
                             bubbleTargets: this,
                             priority: 100,
                             multiple: this.get('multiple'),
-                            loadContent: this.get('loadContent'),
+                            loadContent: true,
```

Comment:
I realize I should have question that again because this makes the `loadContent` completely useless. The content is always needed to display the selection in the selected view and changing that makes the UDW to always provide the content no matter what was the `loadContent` value. So I guess we can just deprecate that parameter: https://jira.ez.no/browse/EZP-26434